xml/Makefile.am

```diff
@@ -21,7 +21,7 @@ dtddir			= $(CRM_DTD_DIRECTORY)
 dtd_DATA		= crm.dtd crm-transitional.dtd
 
 xsltdir			= $(dtddir)
-xslt_DATA		= upgrade06.xsl upgrade-*.xsl
+xslt_DATA		= upgrade06.xsl $(top_srcdir)/xml/upgrade-*.xsl
```

Comment:
Yep, readlink -f is almost identical. realpath is in a separate package on Debian-derived distros, so I wanted to avoid a new build dependency